content/03.results.md

```diff
@@ -38,7 +38,7 @@ More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/ne
 In 2011, the Children’s Brain Tumor Tissue Consortium, now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
 Since the CBTN opened its collection protocol in 2011, before molecular data were integrated into classifications, the majority of the samples within the OpenPBTA lacked molecular subtype annotations at the time of tissue collection.
 Moreover, the OpenPBTA data does not yet feature methylation arrays which are increasingly used to inform molecular subtyping.
-Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to generate research-grade integrated diagnoses for 59% (630/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
+Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to generate research-grade integrated diagnoses for 60% (641/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
```

Comment:
Can you quickly point me to where this update from 630 --> 641 came from?

Reply:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/sample-distribution-analysis/03-tumor-descriptor-and-assay-count.nb.html

Reply:
Ther 59%->60% change should also be made in the discussion, line 13.

Reply:
done! thanks for catching that!